pages/docs/integrations/grouparoo-calculated-property.mdx

```diff
@@ -1,24 +1,94 @@
+</table>
 
-<ValidateAndApplyConfig headingLevel="h3" />
+## Setup
 
-## Calculated Property Next Steps
+To work with the Calculated Property Plugin, you must first install it in an existing Grouparoo project. In addition, you must have at least one other Source configured to make use of Calculated Properties.
 
-Once you have the Plugin installed, App and Source created, and Property configured, you are ready to [validate](/docs/cli/config#validate), [apply](/docs/cli/config#apply), then [import or export your data](/docs/cli/run#run)!
+## App
+
+To create Calculated Properties in Grouparoo, you [create an App](https://www.grouparoo.com/docs/config/apps).
+
+### Options
+
+Some types of Apps require information about how to connect to a service. This integration works a little differently than most and no additional options are required in order to proceed in configuring a Calculated Property as a Source.
+
+## Calculated Property Import
+
+A Calculated Property must be assigned a key, which you can think of as the Property’s name. You must also choose the data type for this property from a dropdown menu and indicate whether this Property is Unique or an Array.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/calculated-property/app/options.png"
+  alt="Calculated Property App Options"
+  width={733}
+  height={211}
+/>
+<br></br>
+<br></br>
+
+### Options
+
+The Calculated Property Source is a specific type of Source. It performs a calculation on properties derived from another source and gives you the ability to generate custom Properties using a JavaScript function in the Property configuration.
+
+Calculations resulting in a string property can be done using a JavaScript pattern similar to the following example for anything that would result in a string Property:
+
+```js
+()=>”{{{lastName}}}{{{firstName}}}”.toLowerCase()
+```
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/calculated-property/app/string-type.png"
+  alt="Calculated Property String Type Example"
+  width={733}
+  height={214}
+/>
+<br></br>
+<br></br>
+
+Calculations using integers need to be parsed as integers, so the pattern for such a calculation would be similar to the following example:
+
+```js
+() =>parseInt(“{{{purchaseCount}}}”)+parseInt(“{{{cartCount}}}”)
+```
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/calculated-property/app/int-type.png"
+  alt="Calculated Property Integer Type Example"
+  width={733}
+  height={258}
+/>
+<br></br>
+<br></br>
+
+## Notes
+
+We strive to keep our documentation accurate, easy to use, and up to date. If you notice that something has changed or is not addressed in the process of setting up your Grouparoo connection with Calculated Properties, please let us know by reaching out to [help@grouparoo.com](mailto:help@grouparoo.com).
+
+As always, if you need support using Grouparoo, reach out to community members in our [Slack channel](https://www.grouparoo.com/chat). We are always happy to help!
```

Comment:
If this is going to be the same on every page, you can make a component for this and share it

Reply:
Basically, yes. I plan to make the component and go through all the docs I have updated in one pass with this.